### ISSUE TITLE
not working since Odyssey 

### ISSUE THREAD

**Issue by LexRottie** [Marginal/StreamSource#6]
07 EDMC will not recognize this as a plugin since Odyssey dropped

**Comment by LILTTALK** [comment]
Download the fork here:

https://github.com/LILTTALK/StreamSource

**Comment by LexRottie** [comment]
Hi has it been updated since i posted?

**Comment by LILTTALK** [comment]
Yes. EDMC change their file structure. It was undated to work with EDMC 5.0.0+

**Comment by Athanasius** [comment]
I've now forked this project to -> https://github.com/Athanasius/EDMC-StreamSource

The 2.0.0-rc1 release there appears to work, but I only performed a perfunctory check of simply logging into the game and checking the output files.

Please direct any further questions to issues **on the fork**, not here.
